pkg/query/paging/request_test.go

```diff
@@ -45,7 +45,8 @@ func TestAddPagingRequest(t *testing.T) {
 			PageSize:  10,
 		}
 
-		sqlMock.ExpectQuery(exactMatchRegexp(`SELECT * FROM "test_objects" WHERE "TheString" = 123 LIMIT $1 OFFSET $2`)).WithArgs(10, 20).WillReturnRows(sqlmock.NewRows([]string{}))
+		sqlMock.ExpectQuery(exactMatchRegexp(`SELECT * FROM "test_objects" WHERE "TheString" = 123 LIMIT $1 OFFSET $2`)).WithArgs(
+			10, 20).WillReturnRows(sqlmock.NewRows([]string{}))
```

Comment:
Does this need modification?

Reply:
Not really. The `make format` cleans up and tidies the repo as part of the `make all` command. 
These are suggestions from `gofumpt` when the command is run.

Reply:
Thanks, just got confused as it seemed a bit random here. But yeah if `make format` was forgotten earlier doing it separetely would be to much hassle.